### ISSUE TITLE
JavaScript Error

### ISSUE THREAD

**Issue by MasterRO94** [laravel/telescope#384]
When I am trying to view the details about a Request I see such errors in the browser console

- [Vue warn]: Error in render: "TypeError: slotProps.entry.content.middleware.join is not a function"
- TypeError: slotProps.entry.content.middleware.join is not a function.

![telescope_issue_1](https://user-images.githubusercontent.com/7365389/48473400-0f0e3200-e801-11e8-98f9-54c77a2dae60.jpg)
![telescope_issue_2](https://user-images.githubusercontent.com/7365389/48473402-0f0e3200-e801-11e8-9cd4-8608c36f63db.jpg)

**Comment by paras-malhotra** [comment]
Try running `php artisan telescope:publish` and check if that resolves the issue

**Comment by riyuk** [comment]
Hey. 
I've installed the latest version. Your fix on 8fa4395c78d32d659b21d6767234b13128f062d3 seems not to actually fix the problem.
Still getting:
```
app.js?id=d31612e65656454934e2:58072 [Vue warn]: Error in render: "TypeError: slotProps.entry.content.middleware.join is not a function"

found in

---> <PreviewScreen> at resources/js/components/PreviewScreen.vue
       <Preview> at resources/js/screens/requests/preview.vue
         <Root>
---
TypeError: slotProps.entry.content.middleware.join is not a function
---
slotProps.entry.content.middleware
              ? _c("tr", [
                  _c("td", { staticClass: "table-fit font-weight-bold" }, [
                    _vm._v("Middleware")
                  ]),
                  _vm._v(" "),
                  _c("td", [
                    _vm._v(
                      "\n            " +
                        _vm._s(slotProps.entry.content.middleware.join(", ")) +
                        "\n        "
                    )
                  ])
                ])
              : _vm._e(),
```

**Comment by MasterRO94** [comment]
@riyuk have you run ‘php artisan telescope:publish’ and cleared browser cache? In my case updating to version 1.0.0 and running these commands removed errors on API requests

**Comment by riyuk** [comment]
Yes i've republished and cleared cache.
In the Database the following values are saved:
`"middleware":{"0":"api","2":"auth:api"}` - if i change that wrong index "2" to "1" it works fine.
Might be a PHP Issue? @themsaid

**Comment by themsaid** [comment]
`gatherMiddleware()` will always return an array. If it's not in your app then please share more details so we can help you debug.

**Comment by riyuk** [comment]
Yes it's a array - but not a sorted array...? The indexes are mixed up.
I've posted the example of my Routes above -- if I do a dd($request->route()->gatherMiddleware()) in the Controller i get the response:
```js
{
  "0": "api",
  "2": "auth:api"
}
```

As I said - the error on Telescope went away after i've fixed the keys on `telescope_entries.content`-Table.

**Comment by themsaid** [comment]
Oh! I can see how this went wrong. Fixing.

**Comment by riyuk** [comment]
Thanks, appreciate it. Question is still: should i also open a issue on laravel/laravel for this?

**Comment by themsaid** [comment]
@riyuk I don't think it's a bug in laravel, it's just how JS reads array with un-sorted or missing keys.
